Xamarin.Forms.Core/VisualElement.cs

```diff
@@ -54,6 +54,65 @@ public partial class VisualElement : Element, IAnimatable, IVisualElementControl
 
 		public static readonly BindableProperty ScaleYProperty = BindableProperty.Create(nameof(ScaleY), typeof(double), typeof(VisualElement), 1d);
 
+		internal static readonly BindableProperty TransformProperty = BindableProperty.Create("Transform", typeof(string), typeof(VisualElement), null, propertyChanged: OnTransformChanged);
+
+		static void OnTransformChanged(BindableObject bindable, object oldValue, object newValue)
+		{
+			if ((string)newValue == "none") {
+				bindable.ClearValue(TranslationXProperty);
+				bindable.ClearValue(TranslationYProperty);
+				bindable.ClearValue(RotationProperty);
+				bindable.ClearValue(RotationXProperty);
+				bindable.ClearValue(RotationYProperty);
+				bindable.ClearValue(ScaleProperty);
+				bindable.ClearValue(ScaleXProperty);
+				bindable.ClearValue(ScaleYProperty);
+				return;
+			}
+				var transforms = ((string)newValue).Split(' ');
+			foreach (var transform in transforms) {
+				if (string.IsNullOrEmpty(transform) || transform.IndexOf('(') < 0 || transform.IndexOf(')') < 0)
+					throw new FormatException("Format for transform is 'none | transform(value) [transform(value) ]*'");
+				var transformName = transform.Substring(0, transform.IndexOf('('));
+				var value = transform.Substring(transform.IndexOf('(') + 1, transform.IndexOf(')') - transform.IndexOf('(') - 1);
+				double translationX, translationY, scaleX, scaleY, rotateX, rotateY, rotate;
+				if (transformName.StartsWith("translateX", StringComparison.OrdinalIgnoreCase) && double.TryParse(value, out translationX))
```

Comment:
Should we have a file with with all the "naming" we care like "translateX and the other css properties?
I know this is something won't change I was more in having a place to see all supported properties, and also if we need to reference them more then once in other places .